main.py

```diff
@@ -0,0 +1,30 @@
+from arguments import parameters
+from keys import check_keys
+from requester import request_weatherapi, request_openmeteo, request_meteostat
+
+# get parameters
+param = parameters()
```

Comment:
This shortcut name somehow bothers me, please change it to either the full name `parameters`, or, even better, `arguments` (I can also live with `args`) since you do have an Argument Parser.

Reply:
done